src/main/frontend/modules/shortcut/config.cljs

```diff
@@ -319,6 +319,10 @@
                                     :fn      (fn [] (state/toggle! :ui/command-palette-open?))
                                     :force?   true}
 
+   :select-graph/open              {:desc    "Open select graph component"
+                                    :fn      (fn [] (state/set-state! :ui/open-select :select-graph))
+                                    :binding "mod+shift+g"}
```

Comment:
I chose 'g' for graph and didn't use :force because it didn't seem necessary

Reply:
@xyhp915 It seems that `:force?` is not used anymore, right?

Reply:
YES. It's unnecessary.

Reply:
@xyhp915 Ok, I'll delete it.